ui/apps/platform/src/Containers/Vulnerabilities/ExceptionManagement/PendingRequests.tsx

```diff
@@ -73,6 +73,7 @@ function PendingApprovals() {
                 {
                     ...searchFilter,
                     'Request Status': ['PENDING', 'APPROVED_PENDING_UPDATE'],
+                    'Expired Request': 'false',
```

Comment:
When a request was canceled, it would still show up in the table. That's because a cancelled request is still set to `PENDING`. The only difference is that it would have the `expired` field set to true. This will help filter out cancelled requests. 

More context on this is explained in this thread: https://redhat-internal.slack.com/archives/C03JMGWJYMD/p1701231468806879